set.mm

```diff
@@ -382517,10 +382525,20 @@ proved using the inner form (see theorem 9.6 of [Schwabhauser] p. 69
 
   ${
     $d a b g i j $.
-    $( Define the relation congruence bewteen shapes.  Definition 4.4 of
-       [Schwabhauser] p. 35.  Ideally, we would define this for functions of
-       any set, but we will used words (functions over ` NN ` ) in most cases.
-       (Contributed by Thierry Arnoux, 3-Apr-2019.) $)
+    $( Define the relation congruence between shapes.  Definition 4.4 of
+       [Schwabhauser] p. 35.  Two shapes are congruent if all corresponding
+       segments between all corresponding points are congruent.  Ideally, we
```

Comment:
I saw in nearby comments the terms "shape" and "sequence". Say somewhere "shapes are finite sequences of points", or maybe simply remove the word "shape" everywhere and only use the term "finite sequence of points" (rather than "word"). @tirix will see what is best.

Reply:
Also: maybe write "...if the segments formed on all pairs of corresponding points..." ?

Reply:
The section header says "series of points"... I think the term "shape" is more intuitive, so we should keep it, but we should define it as "sequence of points" (see the comment for ~iscgrg), as @benjub proposed

Reply:
Good point (hah!). I'll add this text here: **A "shape" is a finite sequence of points.**